tests/standalone/run_all.py

```diff
@@ -281,125 +290,79 @@
 
     extra_flags = ["expect_success", "standalone", "remove_output"]
 
-    if filename == "PySideUsing.py":
+    # specials: these modules need custom checks(implemented below), skip automated checking
+    specials = ['TkInterUsing.py']
+    # skip each test if their respective required import is not present
+    required_m = find_required_module(filename)
+    if required_m and not filename in specials and not hasModule(required_m):
```

Comment:
I think we ought to be able to do without `specials` and just don't `elif` after this.

Typically you use `elif` if you know branches are exclusive, but if they are not you don't mind, and make it two if statements, rather than repeating that knowledge in a condition.

Reply:
I understand, but the issue with TkInter was that the import which should be checked for depends on python version, "Tkinter" for python2 and "tkinter" for python3. In this case without the "specials", a test might be skipped altogether even if the requirement was met, and changing elif to if wouldn't help the issue. If we do this without specials Should we develop a new syntax for the special comment? e.g. `# nuitka-skip-unless: Tkinter || tkinter`, or something along the lines with that?